mysql-test/suite/rpl/t/rpl_slave_dump_thread_wait_sleep_usec.test

```diff
@@ -0,0 +1,19 @@
+--source include/have_debug_sync.inc
+--source include/master-slave.inc
+
+SET @saved_slave_dump_thread_wait_sleep_usec = @@global.slave_dump_thread_wait_sleep_usec;
+SET @@global.slave_dump_thread_wait_sleep_usec= 5000000; # 5 secs
```

Comment:
The test was also changed from the original, which only set the slave_dump_thread_wait_sleep_usec on the slave connection, and not the master connection.